src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/HoverInfoService.cs

```diff
@@ -59,7 +59,7 @@ internal sealed class HoverInfoService : IHoverInfoService
         _htmlFactsService = htmlFactsService;
     }
 
-    public VSInternalHover? GetHoverInfo(RazorCodeDocument codeDocument, SourceLocation location, VSInternalClientCapabilities clientCapabilities)
+    public VSInternalHover? GetHoverInfo(string documentFilePath, RazorCodeDocument codeDocument, SourceLocation location, VSInternalClientCapabilities clientCapabilities)
```

Comment:
❓ Is there a reason not to get the file path from the `RazorCodeDocument`?

Reply:
I, too, thought `RazorCodeDocument` had a file path. Sadly, it does not